### PR TITLE
Fix ansible for Ubuntu 20.04 Focal

### PR DESCRIPTION
**Pull request by ptman**

### CODE REVIEW

**Comment by dessalines** [comment]
@Nutomic Maybe this means it would fix our python3 interpreter problems, and I could remove #859 ?

**Comment by ptman** [comment]
@dessalines I think that's different. I have a python3 based ansible both on ansible machine and the server I'm deploying to.

File: ansible/lemmy.yml [hunk 1]
```diff
@@ -17,6 +18,13 @@
   - name: install dependencies
     apt:
       pkg: ['nginx', 'docker-compose', 'docker.io', 'certbot', 'python-certbot-nginx']
+    when: not (ansible_distribution == 'Ubuntu' and ansible_distribution_major_version == '20')
```
Comment (Nutomic) [inline ansible/lemmy.yml:21]: This is also going to be used for Ubuntu 21 or higher. Shouldnt it use a comparison with "less than" or "greated than"?

**Comment by Nutomic** [comment]
Right I also thought it was about the interpreter errors that @dessalines mentioned. In fact I dont remember any problems with certbot between Ubuntu versions. Did you verify that this works on both 18.04 and 20.04?

File: ansible/lemmy.yml (hunk 1 above)
Comment (ptman) [reply ansible/lemmy.yml]: What are the earliest versions you wish to support? At some point python2 support can just be dropped.

**Comment by ptman** [comment]
I don't have 18.04 to test on.

**Comment by ptman** [comment]
rebased because of conflicts

File: ansible/lemmy.yml [hunk 2]
```diff
@@ -27,7 +28,18 @@
         - 'docker-compose'
         - 'docker.io'
         - 'certbot'
+
+  - name: install certbot-nginx on ubuntu < 20
+    apt:
+      pkg:
         - 'python-certbot-nginx'
+    when: not (ansible_distribution == 'Ubuntu' and ansible_distribution_version|version_compare('20.04', '>=')
```
Comment (Nutomic) [inline ansible/lemmy.yml:36]: Could you please change remove the `not` and use `<` for the comparison?

Btw I noticed this exact problem while migrating dev.lemmy.ml to a new Ubuntu 20.04  server, so this is in fact a necessary change.

**Comment by ptman** [comment]
@Nutomic PTAL

**Comment by Nutomic** [comment]
Thank you :)

File: ansible/lemmy.yml (hunk 1 above)
Comment (Nutomic) [reply ansible/lemmy.yml]: Missed this comment. Not sure tbh, but we can keep it in for at least half a year more, until most people have upgraded their OS.

**Pull request merged**